packages/runner/injection/multi-domain.js

```diff
@@ -40,4 +16,19 @@ const cyBridgeFrame = window.parent.frames[2]
 
 const Cypress = cyBridgeFrame.Cypress
 
+if (!Cypress) {
+  throw new Error('Something went terribly wrong and we cannot proceed. We expected to find the global \
+    Cypress in the spec bridge window but it is missing!. This should never happen and likely is a bug. Please open an issue!')
+}
+
+const timers = createTimers()
```

Comment:
Should we continue to preserve the comment here to explain the timer situation?
```suggestion
// We wrap timers in the injection code because if we do it in the driver (like
// we used to do), any uncaught errors thrown in the timer callbacks would
// get picked up by the top frame's 'error' handler instead of the AUT's.
// We need to wrap the timer callbacks in the AUT itself for errors to
// propagate properly.
const timers = createTimers()
```

Reply:
This comment wouldn't exactly apply to multi-domain but I could say something like:
```
We wrap the timers in the injection code similar to the primary domain.
```
Thoughts?

Reply:
Added comment